multimodal-search-pdf/flows/index.yml

```diff
@@ -0,0 +1,42 @@
+jtype: Flow                                 # this YAML file define the index flow, jtype is the YAML tag
```

Comment:
Being such a large example? should we aim to merge some Executors into CompoundExecutors to have less processes? It may be much more optimal

Reply:
Yes, that makes sense! I will try to add a compressed flow